jbmc/src/java_bytecode/java_types.cpp

```diff
@@ -209,7 +209,7 @@ typet java_type_from_char(char t)
   case 'f': return java_float_type();
   case 'd': return java_double_type();
   case 'z': return java_boolean_type();
-  case 'a': return java_reference_type(void_typet());
+  case 'a': return java_lang_object_type();
```

Comment:
I suppose you want to squash this commit with the last one?

Reply:
Yes was hoping illicit some comments on the original plan - but will squash down and create a fresh PR fixing this would be better.